var/spack/repos/builtin/packages/r-htmltable/package.py

```diff
@@ -35,13 +35,15 @@ class RHtmltable(RPackage):
     compatible with HTML/'LaTeX'."""
 
     homepage = "https://CRAN.R-project.org/package=htmlTable"
-    url      = "https://cran.rstudio.com/src/contrib/htmlTable_1.9.tar.gz"
+    url      = "https://cran.rstudio.com/src/contrib/htmlTable_1.11.2.tar.gz"
     list_url = "https://cran.r-project.org/src/contrib/Archive/htmlTable"
 
+    version('1.11.2', '473e6d486e7714f8dd7f16a31480c896')
     version('1.9', '08c62c19e1ffe570e7d8fa57db5094b9')
 
     depends_on('r-stringr', type=('build', 'run'))
     depends_on('r-knitr', type=('build', 'run'))
```

Comment:
Can you specify the other dependency versions as well? See https://cran.r-project.org/web/packages/htmlTable/index.html